pkg/webhook/alias.go

```diff
@@ -23,12 +23,6 @@ import (
 
 // define some aliases for common bits of the webhook functionality
 
-// CustomDefaulter defines functions for setting defaults on resources.
```

Comment:
Should we keep these aliases and deprecate them as well?

(Just to make this slightly less breaking as it's easily possible)